src/frontend/Sider/Behandling/Venstremeny/Oppsummering/BarnDetaljer.tsx

```diff
@@ -9,17 +9,42 @@ import {
     typeBarnepassTilTekst,
     årsakBarnepassTilTekst,
 } from '../../../../typer/behandling/behandlingFakta/faktaBarn';
-import { JaNei } from '../../../../typer/common';
+import { JaNei, jaNeiTilTekst } from '../../../../typer/common';
 import { tekstEllerKode } from '../../../../utils/tekstformatering';
 
 const BarnDetaljer: React.FC<{ barn: FaktaBarn }> = ({ barn }) => {
     const typePass = barn.søknadgrunnlag?.type;
     const startetIFemte = barn.søknadgrunnlag?.startetIFemte;
+    const utgifter = barn.søknadgrunnlag?.utgifter;
     const årsak = barn.søknadgrunnlag?.årsak;
 
     return (
         <InfoSeksjon label={`${barn.registergrunnlag.navn} ${barn.ident}`} ikon={<ChildEyesIcon />}>
             <BodyShort size="small">{tekstEllerKode(typeBarnepassTilTekst, typePass)}</BodyShort>
+            {utgifter !== undefined && utgifter !== null && (
+                <>
+                    {utgifter.harUtgifterTilPassHelePerioden === JaNei.JA ? (
+                        <>
+                            <BodyShort size="small">
+                                Har utgifter hele perioden:{' '}
+                                {jaNeiTilTekst[utgifter.harUtgifterTilPassHelePerioden]}
+                            </BodyShort>
+                        </>
+                    ) : (
+                        <>
+                            <BodyShort size="small">
+                                Har utgifter hele perioden:{' '}
+                                {jaNeiTilTekst[utgifter.harUtgifterTilPassHelePerioden]}
+                            </BodyShort>
+                            {utgifter.fom && utgifter.tom && (
+                                <BodyShort size="small">
+                                    Har utgifter: {utgifter.fom} - {utgifter.tom}
+                                </BodyShort>
+                            )}
+                        </>
+                    )}
+                </>
+            )}
```

Comment:
Her går det å forenkle litt
* Trenger kun å sjekke `utgifter` "finnes", med `utgifter &&`
* Trenger ikke å sjekke `utgifter.harUtgifterTilPassHelePerioden === JaNei.JA` men kan vise Har utgifter hele perioden direkte

```suggestion
            {utgifter && (
                <>
                    <BodyShort size="small">
                        Har utgifter hele perioden:{' '}
                        {jaNeiTilTekst[utgifter.harUtgifterTilPassHelePerioden]}
                    </BodyShort>
                    {utgifter.fom && utgifter.tom && (
                        <BodyShort size="small">
                            Har utgifter: ${formaterIsoPeriode(utgifter.fom, utgifter.tom)}
                        </BodyShort>
                    )}
                </>
            )}
```